pkg/Makefile

```diff
@@ -70,10 +101,11 @@ tar: prep
 		--license $(LICENSE) \
 		--url $(URL) \
 		--vendor $(VENDOR) \
-		--package $(TARGET_DIR)/aerospike-prometheus-exporter-$(VERSION)-$(ARCH).tar
+		--package $(TARGET_DIR)/$(PKG_FILENAME)_$(VERSION)_$(ARCH).tar
```

Comment:
should use hyphens (-) instead of underscores (_).
naming should be backward compatible and consistent with the rpm style too.